File: SecretProject.BusinessProject/ViewModels/UserData/RegistrationAccountViewModel.cs
using System;

namespace SecretProject.BusinessProject.ViewModels.UserData
{
    [Serializable]
    public class RegistrationAccountViewModel
    {
        public RegistrationAccountViewModel()
        {   
        }

        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
    }
}

File: SecretProject.React/models/Account/registration-account-view-model.ts
export default interface RegistrationAccountViewModel {
    FirstName: string,
    LastName: string,
    Email: string,
    Password: string
    ConfirmationPassword: string
    ConfirmPersonalDataAgreement: boolean
}

export interface RegistrationAccountModal {
    FirstName: string,
    LastName: string,
    Email: string,
    Password: string
}